lib/hydra/works/models/characterization/already_there_strategy.rb

```diff
@@ -4,9 +4,9 @@ class AlreadyThereStrategy < ActiveTriples::ExtensionStrategy
     # Do not add property if the rdf_resource already responds to the property name
     # Do not add property if the rdf_resource already has a property with the same predicate.
     def self.apply(resource, property)
-      return if resource.respond_to?(property.name)
-      return if resource.properties.any? { |p| p[1].predicate == property.predicate }
-      resource.property property.name, property.to_h
+      return if resource.metadata_schema.respond_to?(property.name)
+      return if resource.metadata_schema.properties.any? { |p| p[1].predicate == property.predicate }
+      resource.metadata_schema.property property.name, property.to_h
```

Comment:
I'm of the opinion that `_schema` should stay, because it's describing the schema for description, not the description of any particular object.